specs/source/management/services.yml

```diff
@@ -291,6 +291,7 @@ definitions:
           type: string
       encryptionRequired:
         type: boolean
+        description: Describes whether connections must support end-to-end encryption on both sides of the connection. Read-only property, set at create.
```

Comment:
same comment about it being the create and contradictory to read: "Read-only property, set at create". prolly shoulda commented here